### PR TITLE
Release notes 7.9.0 

### PR DESCRIPTION
**Pull request by karenzone**
Started with generated release notes #12171. Then lots of cleanup happened. 

**PREVIEW:** https://logstash_12179.docs-preview.app.elstc.co/guide/en/logstash/7.9/logstash-7-9-0.html

### CODE REVIEW

**Comment by karenzone** [comment]
@colinsurprenant @jsvd @roaksoax  Here's where we are with 7.9.0 release notes.  Please review.

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -30,6 +31,203 @@ This section summarizes the changes in the following releases:
...
+Resolves https://github.com/elastic/logstash/issues/6366[#6366] and https://github.com/elastic/logstash/issues/8157[#8157].
+* Support using unix pipe as local config file https://github.com/elastic/logstash/pull/11109[#11109]
+* Logging improvements
+** Display Java pipeline initialization time to help with troubleshooting and diagnostics shttps://github.com/elastic/logstash/pull/11749[#11749]
+** Logging framework enhancement to allow more finetuned logging https://github.com/elastic/logstash/pull/11853[#11853]
+** Better logging after definition improvements and script routes in log4j https://github.com/elastic/logstash/pull/11929[#11929] and https://github.com/elastic/logstash/pull/11992[#11992]
+** Improved {ls} startup logging to ensure that 'starting logstash' entry happens before any other log entries https://github.com/elastic/logstash/pull/12086[#12086]
+* Fix: Add back pipelines queue.data and queue.capacity subdocuments for _node/stats https://github.com/elastic/logstash/pull/11923[#11923]
+* Fix: Avoid reloading pipelines that have no changes [#12009]https://github.com/elastic/logstash/pull/12009
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc:144]:
```suggestion
* Fix: Avoid reloading pipelines that have no changes https://github.com/elastic/logstash/pull/12009[#12009]
```

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -30,6 +31,203 @@ This section summarizes the changes in the following releases:
...
+*File Input - 4.2.1*
+
+* Fix: Skip sincedb eviction if read mode completion deletes file during flush https://github.com/logstash-plugins/logstash-input-file/pull/273[#273]  
+* Fix: Watched files performance with huge filesets https://github.com/logstash-plugins/logstash-input-file/pull/268[#268] 
+* Updated logging to include full traces in debug (and trace) levels
+
+*Imap Input - 3.1.0*
+
+* Adds an option to recursively search the message parts for attachment and inline attachment filenames. If the save_attachments option is set to true, the content of attachments is included in the `attachments.data` field. The attachment data can then be used by the Elasticsearch Ingest Attachment Processor Plugin.
+  https://github.com/logstash-plugins/logstash-input-imap/pull/48[#48]
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc:182]:
```suggestion
* Adds an option to recursively search the message parts for attachment and inline attachment filenames. If the save_attachments option is set to true, the content of attachments is included in the `attachments.data` field. The attachment data can then be used by the Elasticsearch Ingest Attachment Processor Plugin https://github.com/logstash-plugins/logstash-input-imap/pull/48[#48]
```

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -30,6 +31,203 @@ This section summarizes the changes in the following releases:
...
+
+
+
+
+
+
+
+
+
+
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc:230]:
```suggestion
```

File: docs/static/releasenotes.asciidoc [hunk 4]
```diff
@@ -30,6 +31,203 @@ This section summarizes the changes in the following releases:
...
+* Refactor code refactor to launch ruby thread from ruby code instead of java (as a workaround for jruby bug) https://github.com/elastic/logstash/pull/11900[#11900]
+* Updates to dependencies
+** Update log4j dependency to 2.13.3
+** Update jruby to 9.2.12.0
+
+==== Plugin releases
+
+*Rubydebug Codec - 3.1.0*
+
+* Replace stale awesome_print library with maintained fork called amazing_print.
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc:157]:
```suggestion
* Replace stale awesome_print library with maintained fork called amazing_print https://github.com/logstash-plugins/logstash-codec-rubydebug/pull/8[#8]
```

File: docs/static/releasenotes.asciidoc [hunk 5]
```diff
@@ -30,6 +31,203 @@ This section summarizes the changes in the following releases:
...
+
+[source,sh]
+-----
+[2020-08-12T14:10:29,388][INFO ][logstash.javapipeline  ][main] Pipeline Java execution initialization time {"seconds"=>0.7}
+-----
+
+
+//TODO: Figure out which featured item this belongs with
+
+* [Featured Item] 40a807d4e - do not call agent.converge_state_and_update before agent.execute (6 weeks ago) <Colin Surprenant> https://github.com/elastic/logstash/pull/12034
```
Comment (roaksoax) [inline docs/static/releasenotes.asciidoc:129]: Hmm, good question. It may belong to PQ improvements, but it is hard to follow given there's too many things linked and nothing really highlights it.
